nearcache/nearcache_it_test.go

```diff
@@ -34,8 +36,120 @@ import (
 
 const (
 	nearCacheDefaultRecordCount = 1000
+	maxCacheSize                = 1
+	maxTTLSeconds               = 2
+	maxIdleSeconds              = 1
 )
 
+func TestSmokeNearCachePopulation(t *testing.T) {
+	// ported from: com.hazelcast.client.map.impl.nearcache.ClientMapNearCacheTest#smoke_near_cache_population
+	tcx := it.MapTestContext{
+		T: t,
+		ConfigCallback: func(tcx it.MapTestContext) {
+			ncc := nearcache.Config{Name: tcx.MapName}
+			ncc.SetInvalidateOnChange(true)
+			tcx.Config.AddNearCache(ncc)
+		},
+	}
+	tcx.Tester(func(tcx it.MapTestContext) {
+		m := tcx.M
+		t := tcx.T
+		ctx := context.Background()
+		const mapSize = 1
```

Comment:
Using 1 in mapSize and maxCacheSize looks ok because we call MapSetOnServer, however such a low number may miss some errors. What about something bigger like 3, 5?

Reply:
The original value for that is 1000, changed that and forgot to set it back: https://github.com/hazelcast/hazelcast-go-client/pull/807/commits/22d0f4598f4b66079464377bdbe5e4aee0db7a09

Reply:
You forgot to update `maxCacheSize`